### ISSUE TITLE
Can I use this lib to move csgo case?

### ISSUE THREAD

**Issue by feeops** [Gobot1234/steam.py#351]
### Summary

Can I use this lib to move csgo case?

### What is the feature request for?

The core library

### The Problem

Can I use this lib to move csgo case?

Like those nodejs package

https://github.com/DoctorMcKay/node-globaloffensive#addtocasketcasketid-itemid

https://github.com/Andrew3019/CSGO-Storage-Unit

https://github.com/nombersDev/casemove

### The Ideal Solution

None

**Comment by Gobot1234** [comment]
Yes, check the docs for Casket.add(). In future please join the discord server (in the readme) or use discussions for questions .
